source/MaterialXGraphEditor/Graph.h

```diff
@@ -210,10 +215,17 @@ class Graph
     bool _delete;
 
     // file dialog information
+#if defined(__APPLE__)
```

Comment:
Hi @dgovil , I'm wondering if it's better to wrap this up in a platform agnostic class and put platform specific code there. Also would be be more consistent to use `zenity` as is done with [NanoGui](https://github.com/mitsuba-renderer/nanogui/blob/f5020e2f3e5114d517642e67afbb21cb88cf04c0/src/common.cpp#L406) ? I'm not sure of the pros / cons of using this but could be used for Mac and Linux, with Windows using specific code.